### PR TITLE
VR-99: converting veritable-ui to esm.

### PR DESCRIPTION
**Pull request by n3op2**
# Pull Request

Ideally merge after -> https://github.com/digicatapult/veritable-ui/pull/37

## Checklist
- [x] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.

## PR Type

Please delete options that are irrelevant.

- [x] Bug Fix
- [x] Feature

## Linked tickets

https://digicatapult.atlassian.net/browse/VR-99

## High level description

Upgrading veritable-ui (this) to be an esm. Currently we are building using older `commonjs`. More details on this -> https://digicatapult.atlassian.net/browse/VR-99

## Operational impact

Low level - currently this services (veritable-ui) not being deployed anywhere.

### CODE REVIEW

File: src/models/__tests__/helpers/mockCompanyHouse.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+import { Env } from '../../../env.js'
 import {
   invalidCompanyNumber,
   noCompanyNumber,
   successResponse,
   validCompanyNumber,
-} from '../fixtures/companyHouseFixtures'
+} from '../fixtures/companyHouseFixtures.js'
 
-const env = container.resolve(Env)
```
Comment (mattdean-digicatapult) [inline src/models/__tests__/helpers/mockCompanyHouse.ts:11]: definitely shouldn't be using `any` here. If you need a type annotation you should be able to use `Env`

File: src/controllers/__tests__/helpers.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import pino from 'pino'
 
 import { Env } from '../../env.js'
 
-export const mockLogger = pino({ level: 'silent' })
+export const mockLogger = pino.default({ level: 'silent' })
```
Comment (mattdean-digicatapult) [inline src/controllers/__tests__/helpers.ts:7]: the `default` here is pretty ugly. You can probably change the import to 

```ts
import { default as pino } from 'pino`
```

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]
Looking good. Couple of small changes needed

File: src/controllers/__tests__/helpers.ts (hunk 2 above)
Comment (n3op2) [reply src/controllers/__tests__/helpers.ts]: So it works for pino, but for pino-http did not work and we use it only in `server.ts` others should have been updated.

File: src/models/__tests__/helpers/mockCompanyHouse.ts (hunk 1 above)
Comment (n3op2) [reply src/models/__tests__/helpers/mockCompanyHouse.ts:11]: ooops:) agreed and it has been updated to be itself.

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**